src/lambda/commands/invokeLambda.ts

```diff
@@ -37,13 +37,20 @@ interface CommandMessage {
     value?: _Blob
 }
 
-export async function invokeLambda(
-    awsContext: AwsContext,
+export async function invokeLambda(params: {
+    /* TODO: Instead of vague scope-leaking objects: awsContext & element, it would be cleaner if this took:
```

Comment:
+1 to this. `awsContext` is left over from the prototype, and we should really get rid of it.